src/components/companies/Companies.tsx

```diff
@@ -0,0 +1,130 @@
+import { useEffect, useState } from "react"
+import MenuBar from "../layout/MenuBar";
+import { Link } from "react-router-dom";
+
+interface CompanyAttributes {
+  id: number;
+  name: string;
+  website: string;
+  street_address: string;
+  city: string;
+  state: string;
+  zip_code: string;
+  notes: string;
+}
+
+interface Company {
+  id: number;
+  type: string;
+  attributes: CompanyAttributes;
+}
+
+function Companies() {
+  const [companies, setCompanies] = useState<Company[] | null>([]); 
+  const [searchTerm, setSearchTerm] = useState("");
+  const [filteredCompanies, setFilteredCompanies] = useState<Company[]>([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchCompanies = async () => {
+      try {
+        const token =
+          "eyJhbGciOiJIUzI1NiJ9.eyJ1c2VyX2lkIjoyLCJyb2xlcyI6WyJ1c2VyIl0sImV4cCI6MTczNDE4ODQ3Nn0.h2YmilmQWZ8CmopONeNZNZpiHIqtdR7M2Siw7jDISaU";
```

Comment:
Do we want to have a token in the code? Even if it isn't a valid one, I think its a potential security risk